frontend/src/store/uam-configure-group/actions.ts

```diff
@@ -32,4 +35,27 @@ const getPermissions = createAsyncThunk<
   return permissions;
 });
 
-export { createGroup, getPermissions };
+const getGroupById = createAsyncThunk<
+  GroupsGetByIdResponseDto,
+  GroupsGetByIdRequestDto,
+  AsyncThunkConfig
+>(ActionType.GET_GROUP, async (request, { extra }) => {
+  const { groupsApi } = extra;
+  const group = await groupsApi.getById(request.id);
+
+  return group;
+});
+
+const updateGroup = createAsyncThunk<
+  GroupsItemResponseDto,
+  GroupUpdateRequestArgumentsDto,
+  AsyncThunkConfig
+>(ActionType.UPDATE_GROUP, async (updatePayload, { extra }) => {
+  const { groupsApi, notification } = extra;
+  const group = await groupsApi.update(updatePayload);
+  notification.success(NotificationMessage.GROUP_UPDATE);
+
+  return group;
+});
+
+export { createGroup, getGroupById, getPermissions, updateGroup };
```

Comment:
```suggestion
export { createGroup, getGroupById, getPermissions, getUsers, updateGroup };
```
we also need to add this action. actions should be per page